bin/compute_mutdensity.py

```diff
@@ -143,8 +145,17 @@ def load_n_process_inputs(maf_path, depths_path, annot_panel_path, sample_name):
     depths_subset_adj_sample_df = depths_df.merge(annot_panel_df.drop_duplicates(subset = ["CHROM", "POS", "REF", "ALT"])[["CHROM", "POS"]],
                                                     on = ["CHROM", "POS"], how = "inner")
 
-    return maf_df, depths_subset_df, depths_subset_adj_df, depths_subset_adj_sample_df
+    # Add domains and exons to maf_df
+    annot_panel_df['CHROM_POS'] = [f'{chrom}:{pos}' for chrom, pos in zip(annot_panel_df['CHROM'], annot_panel_df['POS'])]
+    maf_df_raw['CHROM_POS'] = [mutid.split('_')[0] for mutid in maf_df_raw['MUT_ID']]
```

Comment:
I understand that there is only one mutid in each row, is this right? In this case, maybe we could do:

```suggestion
    maf_df_raw['CHROM_POS'] = maf_df_raw['MUT_ID'].str.split('_')[0]
```

Reply:
Love this suggestion! Vector operations on dataframe make the code run faster!

Reply:
I applied it but it failed and then I had to change it to this:

`maf_df_raw['CHROM_POS'] = maf_df_raw['MUT_ID'].str.split('_').str[0]`

otherwise it was failing because the [0] only took a single item

Reply:
str.split takes an expand argument, then when it set to true it will split the string in columns, then you pick the first one

https://pandas.pydata.org/docs/reference/api/pandas.Series.str.split.html

Reply:
done!